hw/top_earlgrey/data/top_earlgrey.sv.tpl

```diff
@@ -624,6 +624,8 @@ slice = str(alert_idx+w-1) + ":" + str(alert_idx)
     % if m["scan_reset"] == "true":
       .scan_rst_ni  (scan_rst_ni),
     % endif
+
+      // Clock and reset connections
```

Comment:
Super nit, is this comment aligned properly?

Reply:
oops, thanks for noticing, i'll take a look.

Reply:
ah actually this is aligned. The comments are aligned to the 'two spaces' in, but the template code was not.